pkg/rpcclient/nns/record.go

```diff
@@ -33,6 +39,14 @@ const (
 	AAAA RecordType = 28
 )
 
+// ToParameter implements smartcontract.Convertible interface.
```

Comment:
not  ToSCParameter instead of ToParameter?

Reply:
Fixed!